app/javascript/components/zone-form/zone-form.schema.js

```diff
@@ -0,0 +1,95 @@
+import { componentTypes, validatorTypes } from '@@ddf';
+
+const createSchema = (edit) => ({
+    fields: [
+        {
+            component: componentTypes.SUB_FORM,
+            name: 'zone-information-subform',
+            title: __('Zone Information'),
+            fields: [
+                {
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'name',
+                    name: 'name',
+                    label: __('Name'),
+                    isDisabled: edit,
+                    maxLength: 50,
+                    isRequired: true,
+                    validate: [{ type: validatorTypes.REQUIRED }]
+                },
+                {
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'description',
+                    name: 'description',
+                    label: __('Description'),
+                    maxLength: 50,
+                    isRequired: true,
+                    validate: [{ type: validatorTypes.REQUIRED }]
+                },
+                {
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'settings.proxy_server_ip',
+                    name: 'settings.proxy_server_ip',
+                    label: __('SmartProxy Server IP'),
+                    maxLength: 50,
+                }         
+            ]
+        },
+        {
+            component: componentTypes.SUB_FORM,
+            name: 'credentials-subform',
+            title: __('Credentials - Windows Domain'),
+            fields: [
+                {
+                    component: componentTypes.TEXT_FIELD,
+                    id: 'authentications.userid',
+                    name: 'authentications.userid',
+                    label: __('Username'),
+                    maxLength: 50,  
+                },
+                {
+                    component: 'password-field',
+                    edit,
+                    type: 'password',
+                    id: 'authentications.password',
+                    name: 'authentications.password',
+                    label: __('Password'),
+                    maxLength: 50,  
+                },
+                // {
+                //     component: componentTypes.TEXT_FIELD,
+                //     type: 'password',
+                //     id: 'verify',
+                //     name: 'verify',
+                //     label: __('Verify Password'),
+                //     maxLength: 50,
+                // },
```

Comment:
IMO we don't need to verify the password, this is not for creating a password for future use, but to authenticate against windows machines in that zone. There's a high chance that these credentials would be copy-pasted from an external source and if the password was pasted into the first field, it will be pasted into the second one too.

However, if we want to keep this functionality, we should implement the password comparison validation on the DDF level and omit the second field from the submitted values as the API should never care about a password verification.

@h-kataria @himdel wdyt?